e2e-test-app/src/main/java/com/sap/ai/sdk/app/Application.java

```diff
@@ -24,4 +26,14 @@ protected SpringApplicationBuilder configure(final SpringApplicationBuilder appl
   public static void main(final String[] args) {
     SpringApplication.run(Application.class, args);
   }
+
+  /**
+   * This exists only to import ServletException. This import exists only to justify the dependency
+   * jetty-jakarta-servlet-api for the maven-dependency-plugin to succeed.
```

Comment:
(Question)

I don't understand why this is necessary.

Reply:
See above